docs/onboarding/cli.md

```diff
@@ -128,7 +128,7 @@ List mined block hashes
 ```sh
 ironfish miners:mined [START] [STOP]
 ```
-Optional arguments: [START] and [STOP] are either positive numbers that indicate the starting and stopping blocks or are negative to count backwards from the head of the chain.
+Optional arguments: [START] and [STOP] are either positive numbers that indicate the starting and stopping blocks or are negative to count backward from the head of the chain.
```

Comment:
this is changing it to be wrong. Backwards is an adverb

Reply:
In american english it should be backward. American english was used for the rest of the context, hence this correction